src/main/java/de/tum/in/www1/artemis/service/connectors/localci/dto/BuildJobItemReference.java

```diff
@@ -0,0 +1,32 @@
+package de.tum.in.www1.artemis.service.connectors.localci.dto;
+
+import java.io.Serial;
+import java.io.Serializable;
+import java.time.ZonedDateTime;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+/**
+ * Represents a reference to a build job item. This object is used in the build job queue and contains only the necessary
+ * information to identify a build job item. The actual build job item is stored in a Hazelcast IMap. This object is used
+ * to reduce the amount of data used in the Queue.
+ */
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public record BuildJobItemReference(String id, long participationId, long courseId, int priority, ZonedDateTime submissionDate) implements Serializable {
```

Comment:
```suggestion
public record BuildJobItemReferenceDTO(String id, long participationId, long courseId, int priority, ZonedDateTime submissionDate) implements Serializable {
```
The server test currently fail due to this